_includes/header.html

```diff
@@ -5,7 +5,9 @@
 		<title>FOSSASIA | Google Code-In 2017</title>
 
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
+		<!--<link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">-->
+		<link href="css/font-awesome.css" rel="stylesheet" type="text/css" media="all">
+		<link href="css/font-awesome.css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
Please properly align the code, and pls use the min version only.

Reply:
done